docs/eventing/sources/apiserversource/getting-started/README.md

```diff
@@ -81,86 +85,86 @@ Before you can create an API server source, you must install Knative Eventing an
+        kn source apiserver create <apiserversource> \
+          --namespace <namespace> \
+          --mode "Resource" \
+          --resource "Event:v1" \
+          --service-account <service-account> \
+          --sink <sink>
+        ```
+        Where:
+
+        - `<apiserversource>` is the name of the source that you want to create.
+        - `<namespace>` is the name of the namespace that you created in step 1 above.
+        - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
+        - `<sink>` is the name of the PodSpecable object that you want to use as a sink.
+
+    === "YAML"
+
+        ```yaml
+        kubectl create -f - <<EOF
+        apiVersion: sources.knative.dev/v1
+        kind: ApiServerSource
+        metadata:
+         name: <apiserversource>
+         namespace: <namespace>
+        spec:
+         serviceAccountName: <service-account>
+         mode: Resource <!-- should this be a placeholder? -->
+         resources:
+           - apiVersion: v1
+             kind: Event
+         sink:
+           ref:
+             apiVersion: v1
+             kind: Service
+             name: <sink>
+        EOF
+        ```
+        Where:
+
+        - `<apiserversource>` is the name of the source that you want to create.
+        - `<namespace>` is the name of the namespace that you created in step 1 above.
+        - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
+        - `<sink>` is the name of the Knative Service that you want to use as a sink. A Service is used here as an example, however you can use any supported PodSpecable object by updating the `kind` from `Service` to another object type.
+
+
+## Verify the ApiServerSource object
+
+1. Create events by launching a test pod in your namespace by running the command:
 
     ```bash
     kubectl run busybox --image=busybox --namespace=<namespace> --restart=Never -- ls
     ```
-    where;
-    - `<namespace>` is the name of the namespace that you created in step 1 above.
+    Where `<namespace>` is the name of the namespace that you created in step 1 above.
 
-1. Delete the test pod:
+1. Delete the test pod by running the command:
 
     ```bash
     kubectl --namespace=<namespace> delete pod busybox
     ```
-    where;
-    - `<namespace>` is the name of the namespace that you created in step 1 above.
+    Where `<namespace>` is the name of the namespace that you created in step 1 above.
 
-1. View the logs to verify that Kubernetes events were sent to the Knative Eventing system:
+1. View the logs to verify that Kubernetes events were sent to the Knative
+Eventing system by running the command:
 
     ```bash
     kubectl logs --namespace=<namespace> -l app=<sink> --tail=100
     ```
-    where;
+    Where:
+
     - `<namespace>` is the name of the namespace that you created in step 1 above.
     - `<sink>` is the name of the PodSpecable object that you used as a sink in step 5 above.
```

Comment:
I know you haven't touched this in your PR, but can we add a link or description for `PodSpecable` here? Noone knows what that is other than Knative contributors.

Reply:
That makes sense. Do you have a definition or a good link I could use.